New post "FOMO2025" - pull request (#246)
creatis-myriad/creatis-myriad.github.io: collections/_posts/2026-06-04-fomo2025.md

---
layout: review
title: "Towards Brain MRI Foundation Models for the Clinic: Findings from the FOMO25 Challenge"
tags: foundation-models self-supervised-learning MRI medical-imaging segmentation classification regression
cite:
    authors: "A. Munk et al."
    title:   "Towards Brain MRI Foundation Models for the Clinic: Findings from the FOMO25 Challenge"
    venue:   "arXiv preprint arXiv:2604.11679v2 (2026)"
pdf: "https://arxiv.org/pdf/2604.11679"
---

# Introduction

> This paper presents the findings of the [FOMO25 (MICCAI) challenge](fomo25.github.io), a **benchmark**

Reviewer: ```suggestion
This paper presents the findings of the [FOMO25 (MICCAI) challenge](https://fomo25.github.io/), a **benchmark**
```

Evaluation data: validation (20%) and test (80%) datasets were acquired from multiple hospitals in Denmark [3], with MRI sequences and labeling protocols identical to those used for finetuning, but originating from different institutions and geographic regions, thereby constituting an out-of-domain evaluation setting.

![](/collections/images/fomo2025/overview.jpg)

Participants submit inference-only Apptainer containers for evaluation. The organizers provide: baseline implementations, preprocessing code, container validation tools.

<!-- Evaluation on test datasets is performed securely on a private cluster. -->
<!-- 
## Downstream Tasks
### Task 1: Infarct Classification
- Evaluated with AUROC.
### Task 2: Meningioma Segmentation
- Evaluated with Dice and NSD metrics.
### Task 3: Brain Age Regression
- Evaluated with Mean Absolute Error and Pearson correlation.
 -->

# Results

The challenge received:
- 137 registrations
- 18 teams
- 19 evaluated foundation models

For each task, two fully supervised baseline are provided.

> - **Supervised-OOD**, the model is trained on the out-ofdomain finetune dataset (same data as the challenge participants)

Reviewer: ```suggestion
- **Supervised-OOD**, the model is trained on the out-of-domain finetune dataset (same data as the challenge participants)
```

Reviewer 2: I would say "out-of-domain" or "out of domain", both seems to be valid

Reviewer: yes thanks for the correction, i added a space but indeed it was not consistent. I edited my suggested change

SSL pretraining can compensate for severe domain shifts and limited labels.

![](/collections/images/fomo2025/generalization_boost.jpg)

## 2. No single SSL objective dominates all tasks

The authors categorize SSL objectives into:
- Local objectives:
  voxel-level supervision (e.g. MAE).
- Global objectives:
  sequence-level representation learning (e.g. contrastive learning).
- Hybrid objectives:
  combinations of local and global objectives.

![](/collections/images/fomo2025/ssl_objectives.jpg)
*(A–C) Pairwise rank differences between tasks, grouped by SSL objective category (global, hybrid, local): classification vs. segmentation (A), segmentation vs. regression (B), and classification vs. regression (C).*

Observed trends:
- MAE-like local objectives favor segmentation over classification.
- Hybrid objectives favor classification and regression.
- Global objectives surprisingly seem to favor segmentation over classification (unlike natural image segmentation).

Pretraining objectives strongly interact with downstream task characteristics, but none dominates across tasks.

## 3. Scaling laws are weak

> Increasing the **number of pretraining samples** seen does not reliably improve results.The authors suggest that many methods saturate early.

Reviewer: ```suggestion
Increasing the **number of pretraining samples** seen does not reliably improve results. The authors suggest that many methods saturate early.
```

> **Larger encoders** show only weak correlation with downstream performance. Even models with relatively small parameter counts achieve strong rankings.This contrasts with trends in large-scale computer vision and language models.

Reviewer: ```suggestion
**Larger encoders** show only weak correlation with downstream performance. Even models with relatively small parameter counts achieve strong rankings. This contrasts with trends in large-scale computer vision and language models.
```
